index.js

```diff
@@ -55,6 +56,7 @@ var anymatch = function(criteria, value, returnIndex, startIndex, endIndex) {
     if (path.sep === '\\' && typeof string === 'string') {
       altString = string.split('\\').join('/');
```

Comment:
hmm, but using unixify might not be good b/c it strips the drive letter from windows paths (hence the unixify part lol). but if a user is trying to match on a drive letter, it would fail